python/shotgun_model/simple_shotgun_hierarchy_model.py

```diff
@@ -31,7 +31,7 @@ class SimpleShotgunHierarchyModel(ShotgunHierarchyModel):
     can also be subclassed from this class.
     """
 
-    def load_data(self, seed_entity_field, path=None, entity_fields=None):
+    def load_data(self, seed_entity_field, root=None, entity_fields=None):
```

Comment:
No worries. I was just being cautious. All good to push forward with JF's changes. :)